src/styles/blocks/card-block/content-card.scss

```diff
@@ -0,0 +1,134 @@
+.content {
+  margin: 0 auto;
+  display: grid;
+  // gap: 48px;
+  column-gap: 48px;
+  row-gap: 46px;
+  max-width: 944px;
+  padding-top: 50px;
+
+  @media (max-width:488px) {
+    grid-template-columns: 1fr;
```

Comment:
You are using a media query to adjust the grid columns but the task specifies 4 cards per row. Please adjust the grid-template-columns to repeat(4, 1fr) for screens wider than 1024px.